### ISSUE TITLE
[Improvement] Check for softban.

### ISSUE THREAD

**Issue by Acheiro** [NecronomiconCoding/NecroBot#594]
If somebody has softban bot could just take pokestops to check if it's finished or not or just try to catch 1 pokemon per minute. The check could look like: if 3 pokemons fled it can just check pokestops
to not lose rare pokemons if you just teleported in the place it already is. Thanks <3

**Comment by ShawnHardwick** [comment]
I agree there could be better handling and detection for when the account is softbanned.

**Comment by mich3lam** [comment]
Well, leaving the bot alive when softbanned ends the softban... :P

**Comment by wouter2203** [comment]
+ I assume there is no real way to detect softbans.

Yes, you can check if you actually got something from a pokestop. Or if every Pokemon is fleeing you can assume you have a softban, but I do not think there is a clean way to detect a softban. Which is probably why they don't implement this. I would hate it if I had to code based on assumptions.

**Comment by Acheiro** [comment]
@wouter2203 Just gave them an idea and probably a solution after 3 fled pokemons you can be sure you are softbanned
